MANIFEST.in

```diff
@@ -0,0 +1 @@
+recursive-include atool/templates *.html
```

Comment:
Weird, it didn't work when I tried it earlier (it did end up in the built version, though). Probably won't go back and change this but if you care to make it work, feel free.